python/stempy/image/__init__.py

```diff
@@ -653,3 +653,116 @@ def _electron_counted_metadata_to_dict(metadata):
         ret[name] = getattr(metadata, name)
 
     return ret
+
+def virtual_darkfield(array, centers_x, centers_y, radii, plot=False):
```

Comment:
Looks like the `plot` argument isn't actually being used right now?

Reply:
I removed it. It seemed convenient at the time, but hard to implement in practice